website/pages/minification.md

```diff
@@ -172,14 +193,18 @@ In addition to removing defaults, Lightning CSS also omits quotes, whitespace, a
 .foo {
   font-weight: bold;
   background-position: center center;
-  background-image: url("logo.png");
+  background-image: url('logo.png');
 }
 ```
 
 minifies to:
 
 ```css
-.foo{background-image:url(logo.png);background-position:50%;font-weight:700}
+.foo {
+  background-image: url(logo.png);
+  background-position: 50%;
+  font-weight: 700;
+}
```

Comment:
Hmm might need to ignore some of these that are showing exact minified output...

Reply:
Ah haha, yeah that needs an exclude